docs/source/_static/versions.json

```diff
@@ -0,0 +1,20 @@
+[
+    { "name": "latest (main)", "version": "latest", "url": "https://microsoft.github.io/Olive/" },
```

Comment:
This is ambiguous in my opinion. In the drop-down in UI, user will only see "latest (main)". Are we advertising docs for unreleased package? Or is this expected to refer to the docs for latest release (in which case, the next entry should be prefixed with _latest_ and this entry be removed).

Reply:
Call the top entry name as `dev (main)` and version being `main`.

Also, renamed the second entry as `"name": "0.9.1 (latest)"`